android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -42,13 +50,10 @@
 import android.graphics.Rect;
 import android.net.Uri;
 import android.os.Build;
-import android.view.MotionEvent;
-import android.view.View;
 import android.webkit.WebSettings;
 import android.webkit.WebView;
 
-@SuppressWarnings("deprecation")
-public class TiUIWebView extends TiUIView
+@SuppressWarnings("deprecation") public class TiUIWebView extends TiUIView
```

Comment:
formatting still looks off from our standard. annotations should be on the preceding line, not inline as they appear to be throughout your changes.